etc/parameters/default.json

```diff
@@ -718,17 +718,17 @@
     },
     "left_arm": {
       "shoulder_pitch": 1.57,
-      "shoulder_roll": 0.2,
+      "shoulder_roll": 0.0,
       "elbow_yaw": 0.0,
-      "elbow_roll": -0.008,
+      "elbow_roll": 0.0,
       "wrist_yaw": -1.57,
       "hand": 0.0
     },
     "right_arm": {
       "shoulder_pitch": 1.57,
-      "shoulder_roll": -0.2,
+      "shoulder_roll": 0.0,
       "elbow_yaw": 0.0,
-      "elbow_roll": 0.008,
+      "elbow_roll": 0.0,
```

Comment:
same here. You might selected the wrong changes during a rebase? Please revert these changes

Reply:
Yes, reverted